kornia-py/src/warp.rs

```diff
@@ -0,0 +1,42 @@
+use pyo3::prelude::*;
+
+use crate::image::{FromPyImage, PyImage, ToPyImage};
+use kornia_rs::image::Image;
+
+#[pyfunction]
+pub fn warp_affine(
+    image: PyImage,
+    m: (f32, f32, f32, f32, f32, f32),
+    new_size: (usize, usize),
+    interpolation: &str,
+) -> PyResult<PyImage> {
+    // have to add annotation Image<u8, 3>, otherwise the compiler will complain
+    // NOTE: do we support images with channels != 3?
+    let image: Image<u8, 3> = Image::from_pyimage(image)
+        .map_err(|e| PyErr::new::<pyo3::exceptions::PyException, _>(format!("{}", e)))?;
+
+    let new_size = kornia_rs::image::ImageSize {
+        width: new_size.0,
+        height: new_size.1,
```

Comment:
I copied this from `resize.rs`. It was `height: new_size.0, width: new_size.1` originally. Was this a typo? i.e. For the `new_size`, do you use (width, height) or (height, width)?